docs/content/common-subs/required-packages.md

```diff
@@ -60,7 +60,141 @@
         choco install yq -y
         ```
         ``` title="kubectl - https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/ (version range expected: 1.23-1.25)"
-        curl.exe -LO "https://dl.k8s.io/release/v1.27.2/bin/windows/amd64/kubectl.exe"
+        curl.exe -LO "https://dl.k8s.io/release/v1.27.2/bin/windows/amd64/kubectl.exe"    
+        ```  
+    === "WSL with Ubuntu"  
+        ```
+        ```
+        # How to install pre-requisites for a Windows Subsystem for Linux (WSL) envronment using an Ubuntu 22.04.01 distribution
+
+        Tested on a Intel(R) Core(TM) i7-9850H CPU @ 2.60GHz 2.59 GHz with 32GB RAM, a 64-bit operating system, x64-based processor
+        Using Windows 11 Enterprise
+
+        # 1. NB: If you're using a VPN, turn it off
+
+        # 2. Install Ubuntu into WSL
+
+        ## 2.1. In a Windows command terminal run the following to list all the linux distributions that are available online
+        ```
+        wsl -l -o
+        ##2.2 Select a linux distribution and install it into WSL, like this:
+        ```
+        wsl --install -d Ubuntu 22.04.01
+        ```
+        You will see something like this:
+        Installing, this may take a few minutes...
+        Please create a default UNIX user account. The username does not need to match your Windows username.
+        For more information visit: https://aka.ms/wslusers
+        Enter new UNIX username:
+
+        ## 2.3 Enter your new username and password at the prompts, and you will eventually see something like this
+        ```
+        Welcome to Ubuntu 22.04.1 LTS (GNU/Linux 5.10.102.1-microsoft-standard-WSL2 x86_64)
+        ```
+
+        # 2.4 Click on the Windows "Start" icon and type in the name of your distribution into the search box.
+        Your new linux distribution should appear as a local "App". You can pin it to the Windows task bar or to Start for your future convenience.
+        Start a VM using your distribution by clicking on the App.
+
+        # 3. Install pre-requisites into your new VM
+        ## 3.1 update apt-get packages
+        ```
+        sudo apt-get update
+        ```
+        ## 3.2 Install golang
+        ```
+        wget https://golang.org/dl/go1.19.linux-amd64.tar.gz
+        sudo tar -zxvf go1.19.linux-amd64.tar.gz -C /usr/local
+        echo export GOROOT=/usr/local/go | sudo tee -a /etc/profile
+        echo export PATH=$PATH:/usr/local/go/bin | sudo tee -a /etc/profile
```

Comment:
This will break /etc/profile if there are any Windows filesystem paths with spaces or special characters in the system path.
Instead, you need to double quote the expression in the export PATH command:

`echo export PATH="$PATH:/usr/local/go/bin" | sudo tee -a /etc/profile`